NFIQ2/NFIQ2Algorithm/src/nfiq2/modelinfo.cpp

```diff
@@ -31,12 +31,12 @@ NFIQ::ModelInfo::ModelInfo(const std::string &modelInfoFilePath)
 	}
 
 	while (std::getline(fp, line)) {
-		const int eqPos = line.find('=');
+		const std::string::size_type eqPos = line.find('=');
 
 		if (eqPos == std::string::npos) {
 			continue;
 		} else {
-			const unsigned int llen = line.length();
+			const size_t llen = line.length();
```

Comment:
`size_t` here, `size_type` just above. Both are `size_type`. Suggest consistency, or even `auto` so you don't have to worry about it in the future.